src/reference/asciidoc/http.adoc

```diff
@@ -223,6 +222,9 @@ The `expected-response-type` must be compatible with the (configured or default)
 This can be an abstract class or even an interface (such as `java.io.Serializable` when you use Java serialization and `Content-Type: application/x-java-serialized-object`).
 =====
 
+Starting with version 5.5, the `HttpRequestExecutingMessageHandler` exposes an `extractResponseBody` flag (which is `true` by default) to return the whole `ResponseEntity` as a reply message payload or just its body, independently of the provided `expectedResponseType`.
+If body is not present in the `ResponseEntity`, this flag doesn't count and the whole `ResponseEntity` is returned.
```

Comment:
This would be better the other way around.

```suggestion
Starting with version 5.5, the `HttpRequestExecutingMessageHandler` exposes an `extractResponseBody` flag (which is `true` by default) to return just the response body, or to return the whole `ResponseEntity` as the reply message payload, independently of the provided `expectedResponseType`.
If a body is not present in the `ResponseEntity`, this flag is ignored and the whole `ResponseEntity` is returned.
```